airbyte-integrations/connectors/source-google-ads/source_google_ads/state_converter.py

```diff
@@ -0,0 +1,119 @@
+import collections
+from typing import MutableMapping, Any, Tuple, Optional, Union
+
+import pendulum
+from airbyte_cdk.sources.streams.concurrent.cursor import CursorField
+from airbyte_cdk.sources.streams.concurrent.state_converters.abstract_stream_state_converter import ConcurrencyCompatibleStateType
+from airbyte_cdk.sources.streams.concurrent.state_converters.datetime_stream_state_converter import EpochValueConcurrentStreamStateConverter
+from datetime import datetime, timedelta, date
+
+from pendulum import DateTime
+
+from source_google_ads.models import CustomerModel
+
+
+class GadsStateConverter(EpochValueConcurrentStreamStateConverter):
+    zero_value = "0001-01-01"
```

Comment:
Is there some documentation that we can link that indicates that requests should be made for whole days?

Reply:
I don't know where I can find proof for this. But the lowest granularity for google ads api is day